app/src/test/java/com/lastaoutdoor/lasta/data/api/autocomplete/RadarRepositoryImplTest.kt

```diff
@@ -0,0 +1,18 @@
+package com.lastaoutdoor.lasta.data.api.autocomplete
+
+import org.junit.Assert.*
+import org.mockito.Mockito.mock
+import retrofit2.Call
+
+class FakeRadarApiServiceTest : RadarApiService {
+  override fun getSuggestions(
+      query: String,
+      layers: String,
+      country: String,
+      limit: Int
+  ): Call<RadarApiResponse> {
+    return mock(Call::class.java) as Call<RadarApiResponse>
+  }

```

Comment:
Be careful to use mockito library instead of mockk, now we're mixing two mocking libraries amongs our tests